include/mbedtls/config.h

```diff
@@ -3016,7 +3016,7 @@
  * DTLS handshake message reassembly and future message buffering.
  *
  */
-//#define MBEDTLS_SSL_DTLS_MAX_BUFFERING             ( 2 * 16384 )
+//#define MBEDTLS_SSL_DTLS_MAX_BUFFERING             32768
```

Comment:
Please add a comment explaining the value - if it is supposed to be 2 * 16384, for example.

Reply:
Agreed, I will do that in the separate PR I 'fork' from this one.

Reply:
Addressed in https://github.com/hanno-arm/mbedtls/commit/280075104e64a326985417c1e3f7dc4658586c8c.